### ISSUE TITLE
Eslint crashes if import path is broken

### ISSUE THREAD

**Issue by LiSmi** [javierbrea/eslint-plugin-boundaries#96]
Thank you for your work on this project! Just found a minor issue here with broken import paths.

**Describe the bug**

When an import path is broken, eslint crashes entirely (see logs further below). It looks like broken paths resolve to undefined, which then throws an error in picomatch as it expects a string.

**To Reproduce**
1. Configure `boundaries/elements` with some matchers
2. Add a broken import somewhere in your code, eg `import { hello } from '../not/a/real/path'`
3. Run lint to see the error: `TypeError: Expected input to be a string` 

**Expected behavior**
Graceful handling of paths that resolve to undefined. Ideally eslint should not crash, as the crash prevents us from seeing eslint's output (which would include helpful lint errors about the broken import). I'd probably expect broken imports to be ignored by `eslint-plugin-boundaries` as there are other eslint rules which can detect them. (Or failing that, a more specific error message would be useful.)

**Logs**
```
Oops! Something went wrong! :(

ESLint: 7.21.0

TypeError: Expected input to be a string
Occurred while linting /Users/lismi/Documents/project/libs/apollo-mock-utils/src/mock-query.ts:2
    at Function.picomatch.test (/Users/lismi/Documents/project/node_modules/picomatch/lib/picomatch.js:119:11)
    at matcher (/Users/lismi/Documents/project/node_modules/picomatch/lib/picomatch.js:67:50)
    at arrayMatcher (/Users/lismi/Documents/project/node_modules/picomatch/lib/picomatch.js:37:23)
    at Function.micromatch.isMatch (/Users/lismi/Documents/project/node_modules/micromatch/index.js:123:78)
    at isIgnored (/Users/lismi/Documents/project/node_modules/eslint-plugin-boundaries/src/core/elementsInfo.js:22:21)
    at importInfo (/Users/lismi/Documents/project/node_modules/eslint-plugin-boundaries/src/core/elementsInfo.js:167:37)
    at dependencyInfo (/Users/lismi/Documents/project/node_modules/eslint-plugin-boundaries/src/core/dependencyInfo.js:70:22)
    at ImportDeclaration (/Users/lismi/Documents/project/node_modules/eslint-plugin-boundaries/src/rules-factories/dependency-rule.js:24:30)
    at /Users/lismi/Documents/project/node_modules/eslint/lib/linter/safe-emitter.js:45:58
    at Array.forEach (<anonymous>)
```

**Operating system, Node.js and npm versions, and eslint version:**
 - OS: macOS Catalina 10.15.7
 - Node.js: 14.16.0
 - npm: 6.14.11
 - ESLint: 7.21.0

**Comment by javierbrea** [comment]
Hi @LiSmi,
The issue has been fixed in [release v2.1.0](https://github.com/javierbrea/eslint-plugin-boundaries/releases/tag/v2.1.0)

Thanks for your feedback! 🙂

**Comment by LiSmi** [comment]
Thanks so much for such a prompt fix @javierbrea, that's awesome! Works perfectly now 👍
